src/message/messageActionSheet.js

```diff
@@ -230,9 +247,10 @@ export const showActionSheet = (
       ...params,
     });
   };
+  const decorateTitle = title => title.replace('%user%', params.message.sender_full_name);
   showActionSheetWithOptions(
     {
-      options: optionCodes.map(code => _(allButtons[code].title)),
+      options: optionCodes.map(code => decorateTitle(_(allButtons[code].title))),
```

Comment:
Same as above, to use the function for every title where it is only applicable to one seems unwise, don't you think?

Reply:
I agree with you here. The most elegant solution here is to use LocalizableString. The problem is that this must be a string and `_` does not replace variables either.